namespace_opts.go

```diff
@@ -34,7 +34,7 @@ func WithCapabilityIPRanges(ipRanges []IPRanges) NamespaceOpts {
 }
 
 // WithCapabilityBandWitdh adds support for traffic shaping:
-// https://github.com/heptio/cni-plugins/tree/master/plugins/meta/bandwidth
+// https://github.com/containernetworking/plugins/tree/master/plugins/meta/bandwidth
```

Comment:
Does this plugin work correctly today with this library?

There seems to be a mismatch in the json fields

https://github.com/containernetworking/plugins/blob/master/plugins/meta/bandwidth/main.go#L40

and

https://github.com/containerd/go-cni/blob/main/types.go#L50

Reply:
Is this referring to the missing json annotations in the types.go file? I am tracing the code path in the cni,go-cni and contained repos.

Reply:
right should check if all is well after we changed it from int to uint64 here in go-cni.. but kept pointing to hepatio... as a service question